src/lib/ui/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -116,6 +117,13 @@ function AccessCodeRow(props: {
   onClick: () => void
 }) {
   const { onClick, accessCode } = props
+
+  const errorCount = accessCode.errors?.length ?? 0
```

Comment:
Should we show a warning icon if there are any `accessCode.warnings`?

Reply:
Makes sense to me. What do you think @dawnho ?

Reply:
@dawnho will open a new PR if we want to add this feature 👍